PhysicsTools/NanoAOD/python/run3scouting_cff.py

```diff
@@ -334,17 +363,17 @@
 
 ak4ScoutingJetTable = cms.EDProducer("SimplePFJetFlatTableProducer",
       src = cms.InputTag("ak4ScoutingJets"),
-      name = cms.string("ScoutingJet"),
+      name = cms.string("ScoutingCHSJetRecluster"),
       cut = cms.string(""),
-      doc = cms.string("ScoutingJet"),
+      doc = cms.string("ak4 jets from re-clustering scouting PF candidates with CHS applied"),
```

Comment:
Where is CHS applied to `ak4ScoutingJets` and `ak8ScoutingJets`? I see in https://github.com/cms-sw/cmssw/blob/ab08a21a7d4644a6e7c5e79f1857cd464b85cb51/PhysicsTools/Scouting/plugins/Run3ScoutingParticleToRecoPFCandidateProducer.cc#L281 the default value of `CHS` is set to `false`.

Reply:
Thank you for catching this! There seems to be mismatch (whether CHS is applied or not) in private and cmssw code. I will rename them to PFJets.